packages/common/src/helpers/service.ts

```diff
@@ -1,5 +1,5 @@
 import { Cluster } from '@c6o/kubeclient-contracts'
-import { Service } from '@c6o/kubeclient-resources/core/v1'
+import { Namespace, Service } from '@c6o/kubeclient-resources/core/v1'
```

Comment:
Namespace not referenced

```suggestion
import { Service } from '@c6o/kubeclient-resources/core/v1'
```